pkgs/applications/office/onlyoffice-bin/default.nix

```diff
@@ -148,6 +148,7 @@ stdenv.mkDerivation rec {
     ln -s $out/share/desktopeditors/DesktopEditors $out/bin/DesktopEditors
 
     substituteInPlace $out/share/applications/onlyoffice-desktopeditors.desktop \
+      --replace "Icon=onlyoffice-desktopeditors" "Icon=$out/share/desktopeditors/asc-de-128.png" \
```

Comment:
I'm not sure if there a nixpkgs policy on this, but know that .desktop files often get copied out of the nix store and into $HOME/(.config/autostart?) where the nix store path references in them bitrot and break.

I think a better way to handle this is to symlink the icon into a directory where icons are looked for, such as `$out/share/icons/hicolor/64x64/apps/onlyoffice-desktopeditors.png`, instead of changing the .desktop file.

Reply:
Assuming onlyoffice doens't provide icons of multiple sizes, it might be better to put the symlink in `$out/share/pixmaps/onlyoffice-desktopeditors.png` instead.

Reply:
you are right, thanks let me re-commit that